finat/cell_tools.py

```diff
@@ -0,0 +1,8 @@
+import FIAT
+
+
+def max_complex(complexes):
+    """Find the maximal complex in a list of cell complexes.
+    This is a pass-through from FIAT so that FInAT clients
+    (e.g. tsfc) don't have to directly import FIAT."""
+    return FIAT.reference_element.max_complex(complexes)
```

Comment:
This could just be:
```
from FIAT.reference_element import max_complex
```
Maybe with a comment. Then whatever docstring FIAT's `max_complex` has is available.